casework/enrich_title.py

```
@@ -0,0 +1,298 @@
+    current_title = detail.get("title", current_title)
+    court_number = special_court_number(detail)
+
+    if not force and _title_is_valid(current_title, court_number):
+        stats["cases_processed"] -= 1
+        stats["cases_already_valid"] += 1
+        print("  Current title already valid — skipping (use --force)")
+        return
+
+    try:
+        new_title = _generate_title(detail, court_number, invoke_text, usage)
+    except Exception as exc:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)